App/Containers/Cafes.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import { SafeAreaView, View, Text, FlatList } from 'react-native'
+import { connect } from 'react-redux'
+
+import { NavigationScreenProps } from 'react-navigation'
+import { ICafeSession } from '@textile/react-native-sdk'
+import { RootState } from '../Redux/Types'
+
+import { Item, TextileHeaderButtons } from '../Components/HeaderButtons'
+import Separator from '../Components/Separator'
+import Cafe from '../Components/Cafe'
+
+interface StateProps {
+  sessions: ReadonlyArray<ICafeSession>
+}
+
+type Props = StateProps & NavigationScreenProps
+
+class Cafes extends Component<Props> {
+  static navigationOptions = ({ navigation }: NavigationScreenProps) => {
+    const goBack = () => navigation.goBack()
+    const goToCafeRegistration = () => navigation.navigate('RegisterCafe')
+    const headerLeft = (
+      <TextileHeaderButtons left={true}>
+        <Item title="Back" onPress={goBack} iconName="arrow-left" />
+      </TextileHeaderButtons>
+    )
+    const headerRight = (
+      <TextileHeaderButtons>
+        <Item title="Add Cafe" iconName="plus" onPress={goToCafeRegistration} />
+      </TextileHeaderButtons>
+    )
+    return {
+      headerLeft,
+      headerTitle: 'Registered Cafes',
+      headerRight
+    }
+  }
+
+  render() {
+    return (
+      <SafeAreaView>
+        <FlatList
```

Comment:
This flatlist isn't flexing to fill the whole screen. If the number of items is less than the height of the screen, the flatlist is only as tall as the items.

Reply:
Fixed.